### PR TITLE
added mongoDB high availability

### PR DESCRIPTION
**Pull request by LePoulpeOrange**

### CODE REVIEW

File: app-ingress.yaml [hunk 1]
```diff
@@ -3,16 +3,15 @@ kind: Ingress
 metadata:
   name: rocketchat-ingress
 spec:
-  tls:
-  - hosts:
-    - chat.example.com
+   tls:
```
Comment (BastienM) [inline app-ingress.yaml]: Removed the unnecessary blanks (messing up the indentation)

File: README.md [hunk 2]
```diff
@@ -26,11 +26,18 @@ $ cat /path/to/key | base64
 # then past it into data.tls.key
 ```
 
+Edit **MongoDB_HA/storage.yaml** to replace the **google cloud SSD** by solution that you want.
```
Comment (BastienM) [inline README.md]: You are missing a word before the **solution** part.

**Review by BastienM — CHANGES_REQUESTED** [review]

File: app-ingress.yaml (hunk 1 above)
Comment (LePoulpeOrange) [reply app-ingress.yaml]: Done !

File: README.md (hunk 2 above)
Comment (LePoulpeOrange) [reply README.md]: Added word "the"

**Review by BastienM — APPROVED** [review]

**Pull request merged**